### PR TITLE
Wallet dropdown for viewing sol balance, viewing profile, and disconnecting.

### PR DESCRIPTION
**Pull request by echohtp**

### CODE REVIEW

**Comment by echohtp** [comment]
Disconnected 
image

Connected
image

Popup
image

Disconnect hover
image


### Issues
 - Disconnect button uses background color of the element it exists in. I haven't figured out how to apply styles to it yet. 
 - Need to pull balance
 - Need to pull profile image

File: src/pages/index.tsx [hunk 1]
```diff
@@ -147,6 +149,9 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
   })
 
   const { control, watch } = useForm<NFTFilterForm>({})
+  const [showProfilePopover, setShowProfilePopover] = useState(false)
```
Comment (kespinola) [inline src/pages/index.tsx:152]: https://www.radix-ui.com/docs/primitives/components/popover

Radix has a popover can you look to use it?

**Comment by dandlezzz** [comment]
@echohtp can we get this in?

**Comment by kespinola** [comment]
handled in this PR.

https://github.com/holaplex/marketplace/pull/77

**Pull request closed without merging**